<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRCWProjectionDimPowerwallClass</class>
 <widget class="QWidget" name="VRCWProjectionDimPowerwallClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRCWProjectionDim</string>
  </property>
  <layout class="QVBoxLayout" name="vrcwProjectionDimPowerwallClassVLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>66</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="headerWidgetHLayout">
      <item>
       <layout class="QHBoxLayout" name="headerHLayout">
        <item>
         <layout class="QVBoxLayout" name="headingVLayout">
          <item>
           <widget class="QLabel" name="header">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Projection Configuration - Powerwall - Dimensions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="description">
            <property name="text">
             <string>Resolution - Size - Viewer position - Floor height</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="headerVSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="headerHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="headerLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="mainWidgetVLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="dimTiledWidget" native="true">
        <layout class="QVBoxLayout" name="dimTiledWidgetVLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="dimWidget" native="true">
        <layout class="QVBoxLayout" name="dimWidgetVLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="posWidget" native="true">
        <layout class="QVBoxLayout" name="posWidgetVLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
